src/tests/UserInputBlock.test.tsx

```diff
@@ -88,6 +88,33 @@ test('UserInputBlock requires non-empty input', async () => {
   expect(screen.queryByText('Please provide valid input')).toBeNull()
 })
 
+test('UserInputBlock input isnt over the max length', async () => {
+  render(
+    <UserInputBlock content="<p>Content text</p>" prompt="<p>Prompt text</p>" ack="<p>Ack text</p>" buttonText="CustomButtonText"/>
+  )
+
+  await act(async () => {
+    screen.getByRole('button').click()
+  })
+
+  screen.getByText('Content text')
+  screen.getByText('Prompt text')
+  screen.getByRole('textbox')
+  screen.getByRole('button')
+  screen.getByText('Please provide valid input')
+
+  await act(async () => {
+    fireEvent.change(screen.getByRole('textbox'), { target: { value: new Array(MAX_CHARACTER_INPUT_BLOCK_LENGTH + 2).join('a') } })
```

Comment:
Out of curiosity: Is there a specific reason why this test case uses a different idiom from the others where you employ `String.repeat()`?

Reply:
Nope - Just a typo, I'll change to the other version. I think it's cleaner